bundle/edu.gemini.itc.web/src/test/scala/edu/gemini/itc/web/baseline/BaselineTest.scala

```diff
@@ -26,7 +26,7 @@ import org.junit.{Ignore, Test}
  */
 class BaselineTest {
 
-  @Ignore
+  //@Ignore
```

Comment:
This should just be commented out to create the `baseline.txt` file: once you've done that, you should uncomment it.

Reply:
I started working on a new feature, and those changes were included in this commit. I think I need to cancel this pull request and start from scratch.